app/models/rook.rb

```diff
@@ -1,2 +1,19 @@
 class Rook < ChessPiece
-end
\ No newline at end of file
+  def display
+    if color == "black"
+      return "&#9820;"
+    else
+      return "&#9814;"
+  end
+   
+  
+  
+ def valid_move?(board, stop)
+  if self.is_vertical_move?(board,stop)
+    return true
+  elsif self.is_horizontal_move?(board, stop)
+    return true
+  end
```

Comment:
If we're going to use an if statement with an elsif, there should be an else statement as well. Currently in the case that the move is not vertical or horizontal, the loop will go to the third option (else) which isn't specified so it'll default to nil which we don't want sent back to whatever method is calling this one.
Alternatively, this method could be rewritten with the single line as:

```
def valid_move?(board, stop)
  self.is_vertical_move?(board, stop) || self.is_horizontal_move?(board, stop)
end